stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,1023 @@
+                        <span class="fd-checkbox__text fd-list__title">
+                            Tomato
+                        </span>
+                    </label>
+                </li>
+            </ul>
+        </div>
+        </div>
+    </div>
+</div>
+`;
+
+grouping.parameters = {
+    docs: {
+        storyDescription:
+            'In cases where the list items need to be categorized into groups, it is possible to add headers for each category as seen below.'
+    }
+};
+
+export const matchPopoverBodySize = () => `<div style="height: 300px;">
+    <div class="fd-popover">
+        <div class="fd-popover__control" role="combobox" aria-controls="F4HGFHX34a" aria-expanded="true" aria-haspopup="true">
+            <div class="fd-input-group fd-input-group--control" tabindex="0">
+                <div class="fd-tokenizer">
+                    <div class="fd-tokenizer__inner">
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Apple
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Orange
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Kiwi
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-tokenizer__indicator">2 more</span> 
+                        <input class="fd-input fd-input-group__input fd-tokenizer__input"  aria-label="multi input sample popover size body"/>
+                    </div>
+                </div>
+                <span class="fd-input-group__addon fd-input-group__addon--button">
+                    <button class="fd-input-group__button fd-button fd-button--transparent" aria-label="down arrow" aria-controls="F4HGFHX34a" aria-expanded="false" aria-haspopup="true">
+                        <i class="sap-icon--navigation-down-arrow"></i>
+                    </button>
+                </span>
+            </div>
+        </div>
+        <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown" aria-hidden="false" id="F4HGFHX34a">
+        <div class="fd-popover__wrapper">
+            <ul class="fd-list fd-list--multi-input"  role="listbox" aria-multiselectable="true" aria-label="selectMultipleFruitsLabel">
```

Comment:
These do not have a visible group label that is being referred to by id, and thus the label should be added directly.
```suggestion
            <ul class="fd-list fd-list--multi-input"  role="listbox" aria-multiselectable="true" aria-label="Fruits">
```